lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -61,7 +64,7 @@ Value RequestCertificateHandler(const MessageOrigin::Ptr& origin, const Dictiona
 		logmsg << "Received certificate request for CN '" << cn << "'";
 
 		try {
-			signedByCA = VerifyCertificate(cacert, cert, listener->GetCrlPath());
+			signedByCA = VerifyCertificate(cacert, cert, listener->GetCrlPath(), chain);
```

Comment:
And the only one making use of any of the above is RequestCertificateHandler?

I have to re-check https://github.com/Icinga/icinga2/pull/8859#issuecomment-1551364930 by myself...